sklearn/ensemble/gradient_boosting.py

```diff
@@ -934,6 +935,12 @@ def fit(self, X, y, sample_weight=None, monitor=None):
             computing held-out estimates, early stopping, model introspect, and
             snapshoting.
 
+        presort : bool, optional (default=False)
```

Comment:
Should it be true by default?
Is there any benefits in letting the user choose?